registry/virt/0.14.0/virtoperator.clusterrolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
All of the virtoperator stuff should come from kubevirt/kubevirt but that automated process can be dealt with later IMO.